public/app/plugins/datasource/stackdriver/components/Project.tsx

```diff
@@ -1,31 +1,52 @@
 import React from 'react';
-import { Input } from '@grafana/ui';
 import StackdriverDatasource from '../datasource';
+import { MetricSelect } from 'app/core/components/Select/MetricSelect';
 
 export interface Props {
   datasource: StackdriverDatasource;
+  onChange: (project: any) => void;
+  defaultProject: string;
 }
 
-interface State {
-  projectName: string;
+export interface State {
+  value: string;
+  projectList: string[][];
 }
 
 export class Project extends React.Component<Props, State> {
   state: State = {
-    projectName: 'Loading project...',
+    projectList: [],
+    value: '',
   };
 
   async componentDidMount() {
-    const projectName = await this.props.datasource.getDefaultProject();
-    this.setState({ projectName });
```

Comment:
How is this supposed to work?

Previously, the defaultProject for that service account was loaded from the backend. Now that we only load the project list, and don't set a value on that dropdown, it will break all the queries/dashboards for all users that have GCE configured. 

I think either the project list needs to return the default account, or we need to fire off a second query to the backend to retrieve the default account and set that as value for the project dropdown.

Reply:
For this one I need @eliaslaouiti but isn't available until next week. We understand the problem